.gitignore

```diff
@@ -2,3 +2,6 @@
 *.class
 *.jar
 
+.idea/
+*.iml
+/classes/
```

Comment:
for what is this line?

Reply:
ignoring all the files that Intellij idea generates to function correctly. this files are not a direct part of the project, thus they shall be ignored